packages/vuetify/src/components/VFeatureDiscovery/VFeatureDiscovery.ts

```diff
@@ -310,6 +310,9 @@ export default baseMixins.extend<options>().extend({
         'aria-hidden': !this.isActive,
       }
     },
+    closeConditional (): boolean {
+      return !this.persistent && this.isActive
```

Comment:
should this fix be applied to this PR as well? https://github.com/vuetifyjs/vuetify/commit/2ad27f604fb1e3f228224bf6e046c81ec067eca8

Reply:
idk, but maybe

Reply:
VMenu and menuable mixin was used as a base for its opening/closing/positioning, so probably yes. But some code has been removed because Feature Discovery has a different props for its positioning system. So, it depends on what thriggers the issue.